teams_rules.go

```diff
@@ -35,6 +35,14 @@ type TeamsRuleSettings struct {
 
 	// whether to disable dnssec validation for allow action
 	InsecureDisableDNSSECValidation bool `json:"insecure_disable_dnssec_validation"`
+
+	EgressSettings *EgressSettings `json:"egress"`
+}
+
+type EgressSettings struct {
+	Ipv6Range    string `json:"ipv6"`
+	Ipv4         string `json:"ipv4"`
+	Ipv4Fallback string `json:"ipv4_fallback"`
```

Comment:
```suggestion
	IPv6Range    string `json:"ipv6"`
	IPv4         string `json:"ipv4"`
	IPv4Fallback string `json:"ipv4_fallback"`
```

any reason why the struct field is IPv6Range and not IPv6 given the JSON key?

Reply:
The ipv6 field does indeed have a cidr, the ipv4 does not. I think the json key was renamed at someones request for brevity on the wire 🤷